includes/widgets/class-wc-widget-layered-nav.php

```diff
@@ -213,16 +215,26 @@ protected function get_current_term_slug() {
 	 * @return bool Will nav display?
 	 */
 	protected function layered_nav_dropdown( $terms, $taxonomy, $query_type ) {
+		global $wp;
 		$found = false;
 
 		if ( $taxonomy !== $this->get_current_taxonomy() ) {
 			$term_counts          = $this->get_filtered_term_product_counts( wp_list_pluck( $terms, 'term_id' ), $taxonomy, $query_type );
-			$_chosen_attributes   = WC_Query::get_layered_nav_chosen_attributes();
+			$_chosen_attributes   = WC_Query                                                                                       ::get_layered_nav_chosen_attributes();
```

Comment:
Not sure if this even will work... Looks like that is missing more than just `;`.